swarm/network/protocol.go

```diff
@@ -35,8 +35,6 @@ import (
 
 const (
 	DefaultNetworkID = 3
-	// ProtocolMaxMsgSize maximum allowed message size
-	ProtocolMaxMsgSize = 10 * 1024 * 1024
```

Comment:
This is not used, but both the `stream` protocol and the `bzz` protocol have a `MaxMsgSize` calculated the same way.

Maybe we should remove the individual calculations and use this constant instead

Reply:
I don't know if it makes sense abstracting the `stream` and `bzz` protocol message size. The fact that it is the same max size right now, doesn't mean that it will always stay that way, or that these are one and the same thing.

Let's decide in future PR if we want to do that.